test/self/formats.py

```diff
@@ -49,3 +49,37 @@ def test_frame_reader(width, height, fmt):
     for n in range(1, frames):
       y, u, v = FrameReaders[fmt](fd, width, height)
       assert fd.tell() == size * n
+
```

Comment:
Can you add "Y210" to the `__MYPARMS__` so that the other existing tests can check your code, too?

You can generate the necessary test assets for Y210, for example:
```
gst-launch-1.0 videotestsrc num-buffers=1 ! video/x-raw,format=Y210,width=176,height=144 ! checksumsink2 dump-output=1 dump-location=176x144.Y210
```

Reply:
ok, let me to check on this